Validation/CaloTowers/test/macros/RelValHarvest_2023.py

```diff
@@ -48,55 +48,61 @@ def getDataSets( dsFlags = {'RelValMinBias_13__':'MinBias'},
                 if str in line:
                     # extract dataset path
                     path = line.split('\'')[1].strip()
-                    #print ("Getting DQM output from dataset: %s"%path)
-                    if (path.find("Ideal") > 0 or path.find("design") > 0 or path.find("FastSim") > 0 or path.find("DQM") < 0 or path.find("Pixel") > 0 ):  #skip for unnecessary samples
-                        continue
-                    print (path.split("/")[-1]) #path
-                    if printDS:
-                        continue
+                    #print("Getting DQM output from dataset: %s"%path)
+                 #   if (path.find("Ideal") > 0 or path.find("design") > 0  or path.find("DQM") < 0 or path.find("Pixel") > 0 or path.find("2021") > 0 or path.find("FastSim") > 0 or path.find("asymptotic") > 0):  #skip for unnecessary samples
+                    if (path.find("2023") < 0):
+                        continue    
+                        print(path.split("/")[-1]) #path
+                        if printDS:
+                           continue
 
                     # construct file name
                     fname = path.split("/")[-1]
-
+                    #print "smDebug2: ",fname
                     # create file name for use with hcal scripts
                     info = fname.split("__")[2].replace(label, "").strip("-")
-
+                    #print "smDebug3: ",info
```

Comment:
Hi @abdoulline , Do you want to remove such lines? If to be retained may write in the proper format, please.